integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -23,84 +31,211 @@
+    catalogsPage.setCatalogFixedProp("uri", hiveMetastoreUri);
+    // set iceberg warehouse
+    catalogsPage.setCatalogFixedProp("warehouse", hdfsUri);
+    clickAndWait(catalogsPage.handleSubmitCatalogBtn);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(ICEBERG_CATALOG_NAME));
+  }
+
+  @Test
+  @Order(3)
+  public void testCreateMysqlCatalog() throws InterruptedException {
+    // create mysql catalog actions
+    clickAndWait(catalogsPage.createCatalogBtn);
+    catalogsPage.setCatalogNameField(MYSQL_CATALOG_NAME);
+    // select provider as mysql
+    clickAndWait(catalogsPage.catalogProviderSelector);
+    catalogsPage.clickSelectProvider("jdbc-mysql");
+    catalogsPage.setCatalogCommentField("mysql catalog comment");
+    // set mysql catalog props
+    catalogsPage.setCatalogFixedProp("jdbc-driver", MYSQL_JDBC_DRIVER);
+    catalogsPage.setCatalogFixedProp("jdbc-url", mysqlUri);
+    catalogsPage.setCatalogFixedProp("jdbc-user", COMMON_JDBC_USER);
+    catalogsPage.setCatalogFixedProp("jdbc-password", COMMON_JDBC_PWD);
+    clickAndWait(catalogsPage.handleSubmitCatalogBtn);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(MYSQL_CATALOG_NAME));
+  }
+
+  @Test
+  @Order(4)
+  public void testCreatePgCatalog() throws InterruptedException {
+    // create postgresql catalog actions
+    clickAndWait(catalogsPage.createCatalogBtn);
+    catalogsPage.setCatalogNameField(PG_CATALOG_NAME);
+    // select provider as mysql
+    clickAndWait(catalogsPage.catalogProviderSelector);
+    catalogsPage.clickSelectProvider("jdbc-postgresql");
+    catalogsPage.setCatalogCommentField("postgresql catalog comment");
+    // set mysql catalog props
+    catalogsPage.setCatalogFixedProp("jdbc-driver", PG_JDBC_DRIVER);
+    catalogsPage.setCatalogFixedProp("jdbc-url", postgresqlUri + ":5432/" + PG_JDBC_DB);
+    catalogsPage.setCatalogFixedProp("jdbc-user", COMMON_JDBC_USER);
+    catalogsPage.setCatalogFixedProp("jdbc-password", COMMON_JDBC_PWD);
+    catalogsPage.setCatalogFixedProp("jdbc-database", PG_JDBC_DB);
+
+    clickAndWait(catalogsPage.handleSubmitCatalogBtn);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(PG_CATALOG_NAME));
+  }
+
+  @Test
+  @Order(5)
+  public void testCreateFilesetCatalog() throws InterruptedException {
+    clickAndWait(catalogsPage.createCatalogBtn);
+    catalogsPage.setCatalogNameField(FILESET_CATALOG_NAME);
+    clickAndWait(catalogsPage.catalogTypeSelector);
+    catalogsPage.clickSelectType("fileset");
+    catalogsPage.setCatalogCommentField("fileset catalog comment");
+    clickAndWait(catalogsPage.handleSubmitCatalogBtn);
+    Assertions.assertTrue(catalogsPage.verifyGetCatalog(FILESET_CATALOG_NAME));
+  }
+
+  @Test
+  @Order(6)
   public void testRefreshPage() {
     driver.navigate().refresh();
-    Assertions.assertEquals(driver.getTitle(), "Gravitino");
+    Assertions.assertEquals(driver.getTitle(), WEB_TITLE);
     Assertions.assertTrue(catalogsPage.verifyRefreshPage());
   }
```

Comment:
The `verifyRefreshPage` is too simple, need to verify expended tree node and correct table values too. @ch3yne

Reply:
I added verified created catalogs.